src/styles/blocks/stars.scss

```
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--1 :first-child,
```

```suggestion
  &--1 :nth-child(-n + 1,
```

Make all code the same style